charts/azuremonitor-containers/values.yaml

```diff
@@ -71,6 +71,10 @@ omsagent:
     ## Applicable for only Azure Stack Edge K8s since it has custom mount path for container logs which will have symlink to /var/log path
     custommountpath: ""
 
+  arcasettings:
+    isArcACluster: false
+    metricsEndpoint: <your_metrics_endpoint>
```

Comment:
Hi, @MSFTXiangyu ,can these set by Arc k8s extension platform  for ASZ cluster similar to other fields set here https://github.com/microsoft/Docker-Provider/blob/ci_dev/charts/azuremonitor-containers/values.yaml#L8?

Reply:
Do you mean place these "arcasettings" under "Azure" section?